script.js

```diff
@@ -0,0 +1,33 @@
+document.addEventListener("DOMContentLoaded", () => {
+  const sections = document.querySelectorAll("main section");
+  const navLinks = document.querySelectorAll(".nav__links__list a");
+
+  function highlightNavLink() {
+    const scrollPosition = window.scrollY;
+
+    sections.forEach((section) => {
+      if (
+        (scrollPosition >= section.offsetTop - 100) &&
+        scrollPosition < section.offsetTop + section.offsetHeight
+      ) {
+        const sectionId = section.getAttribute("id");
+
+        navLinks.forEach((navLink) => {
```

Comment:
![image](https://github.com/peviitor-ro/oportunitatisicariere/assets/120104711/398adacc-362e-4516-b22c-cd1f03f74cca)

You need to change a bit the values here since it's showing wrong for the current section I'm in - check the image.